keyboards/geistmaschine/geist/info.json

```diff
@@ -0,0 +1,273 @@
+{
+    "manufacturer": "Geistmaschine",
+    "keyboard_name": "Geist",
+    "maintainer": "ebastler",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "encoder": true
```

Comment:
If encoders are enabled at the keyboard level, there should be some functionality for them at the keyboard level as well. See [doc](https://docs.qmk.fm/#/feature_encoders?id=callbacks).

Reply:
Hmm, how do I go about doing that? If I have encoder callbacks defined in board.c, will that not interfere with the mappings in keymap.c? I figured enabling two different encoder mappings is not allowed. Would I add #ifndef around the callback for that?

Reply:
Encoder map will take precedence automatically without any guards. But if you did want to make it more intuitive...
```c
#if defined (ENCODER_ENABLE) && !defined (ENCODER_MAP_ENABLE)
bool encoder_update_kb(uint8_t index, bool clockwise) {
//...
}
#endif

Reply:
Perfect! I'll add the encoder update kb function tomorrow, and apply the same to my other PR as well. Thank you very much for the info - I was not aware it would automatically prefer the keymap settings.

Reply:
Hope I did it correctly - once you signed this one off (or tell me if I did something wrong) I'll apply the changes to my other PR as well.

Reply:
Yep, looks good to me.